server/pkg/pipeline/queue.go

```diff
@@ -43,11 +52,20 @@ func NewQueue[T any](driver QueueDriver[T], itemProcessor QueueItemProcessor[T])
 		workerPool:    pond.New(QueueWorkerCount, QueueWorkerBufferSize),
 	}
 
+	if namedDriver, ok := driver.(namedDriver); ok {
+		queue.name = namedDriver.Name()
+	}
```

Comment:
This was the way I found to be able to populate the queue name from the `postgresQueueDriver channel`.